### ISSUE TITLE
Lag when pressing and holding local change speed hotkeys

### ISSUE THREAD

**Issue by orkhan10** [polywock/globalSpeed#140]
There is a lag when I press and hold local change speed shortcuts. Lets say your change interval is +0,1 for the shortcut. When you press once, reaction is immediate. When you press and hold it applies first 0,1s but needs 1-2 seconds to start applying increment continuously.

I updated to Monterey. I don't know whether that is the cause or not. But this was not the normal behaviour before.

Could someone confirm whether this is the normal behaviour? 
Any idea how to return the previous behaviour?

**Comment by polywock** [comment]
Hello. That's the intended behavior. Every few seconds it checks all videos to make sure they're at the intended speed. But, for performance reasons, I made it so this interval function only runs on active tabs. So if you have 100 tabs, it will just only run the interval function on the open tab. 

While in local mode (pinned), your speed changes only apply to your current tab (active). That means the interval function would apply the speed change in a few seconds. 

While in global mode (unpinned), your speed changes has to apply to all global tabs. But, the interval function isn't running on background tabs so I have to apply the speed change a different way (which happens to be immediate). 

Short version: A different approaches for global and local mode. 

I'll make it immediate for local mode as well in the next update.

**Comment by orkhan10** [comment]
Thanks a lot @polywock for taking it into consideration. 🙏
